plugins/modules/system/seport.py

```diff
@@ -89,6 +94,14 @@
     proto: tcp
     setype: memcache_port_t
     state: present
+
+- name: Remove tcp port 22 local modification if exists
+  local_seport:
+    ports: 22
+    protocol: tcp
+    setype: http_port_t
+    state: absent
+    local: yes
```

Comment:
not sure what is the convention for these docs, but the recommendation now is to use `true` and `false` instead of `yes` and `no`

Reply:
Fixed, thanks